src/context/AuthContext.ts

```diff
@@ -11,11 +12,15 @@ const authReducer = (state: StateType, action: ActionType) => {
   }
 };
 
-const signIn = (dispatch: any) => async (payload: { email: string, password: string }) => {
+const signIn = (dispatch: React.Dispatch<ActionType>) => async (payload: { email: string, password: string }) => {
   const { token, tokenExpireDate } = await Authentication.authenticate(payload);
   await asyncStorage.setAlenviToken(token, tokenExpireDate);
 
   dispatch({ type: 'signin', payload: { alenviToken: token } });
 };
 
-export const { Provider, Context } = createAuthContext(authReducer, { signIn }, { alenviToken: null });
+export const { Provider, Context } = createAuthContext(
+  authReducer,
+  { signIn },
+  { alenviToken: null, signIn: async () => {} }
```

Comment:
c'est pour corriger l'erreur ts ? tu peux m'expliquer en quoi ?

Reply:
Enfait les fonctions du contexte font partie du state. Du coup je les ai ajouté dans les StateType et il faut leur donner une valeur par defaut. 
(un des liens qui m'ont aidé : https://dev.to/alexander7161/react-context-api-with-typescript-example-j7a)